services/app-api/utils/validation/validation.ts

```diff
@@ -2,6 +2,22 @@ import * as yup from "yup";
 import { nested, endDate, schemaMap } from "./schemaMap";
 import { AnyObject } from "../types/types";
 import { error } from "../constants/constants";
+import { APIGatewayEvent } from "aws-lambda";
+
+// validate report keys
+export const validReport = (
```

Comment:
Suggestion: When referenced in other files, `validReport` may seem like it's an actual report instead of a boolean representing whether an event has all necessary path parameters. Maybe we could rename `checkEventParams` or `hasEventParams` or something to be more clear about what this check is.

Reply:
Note: It's also worth noting that this is only relevant for `/report/` endpoints, and will not work for any other API handler.